torchao/dtypes/floatx/cutlass_semi_sparse_layout.py

```diff
@@ -80,6 +112,7 @@ def __init__(
         self.meta = meta
         self.scale = scale
         self._layout = _layout
+        self._shape = shape
```

Comment:
why do we need a separate shape, instead of deriving from the `sparse` tensor?

Reply:
nit: also sparse naming might be a bit vague, renaming to `sparse_data` or something might be better